src/app/server/Mdns.cpp

```diff
@@ -116,19 +116,18 @@ CHIP_ERROR AdvertiseOperational()
         {
             uint8_t mac[8];
 
-            const auto advertiseParameters =
-                chip::Mdns::OperationalAdvertisingParameters()
-                    .SetPeerId(PeerId().SetFabricId(fabricInfo.GetFabricId()).SetNodeId(fabricInfo.GetNodeId()))
-                    .SetMac(FillMAC(mac))
-                    .SetPort(GetSecuredPort())
-                    .SetMRPRetryIntervals(CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL,
-                                          CHIP_CONFIG_MRP_DEFAULT_ACTIVE_RETRY_INTERVAL)
-                    .EnableIpV4(true);
+            const auto advertiseParameters = chip::Mdns::OperationalAdvertisingParameters()
+                                                 .SetPeerId(fabricInfo.GetPeerId())
```

Comment:
Based on the code before this PR, this is probably functionally correct.  However, why are we advertising something called peer ID?  That makes no sense.

Reply:
Discussed offline.  This object name (PeerId) might be confusing, but is preexisting.  Maybe in the future we can come up with a better name to describe what is in essence a fully-qualified node identity, which is obviously different than NodeId.